client/src/Components/Card/EmissionsCard.js

```diff
@@ -32,4 +22,4 @@ class EmissionsCard extends Component {
   }
 }
 
-export default graphql(getEmissionsQuery)(EmissionsCard);
\ No newline at end of file
+export default EmissionsCard;
```

Comment:
Files should end with a new line. https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files

Reply:
Thanks for the professional insight. I didn't know.